frontend/packages/component-library/src/cms/iconHighlight/IconHighlight.tsx

```diff
@@ -0,0 +1,97 @@
+import classNames from 'classnames';
+import React, {Key, ReactNode, HTMLAttributes} from 'react';
+
+import FontAwesomeV6Icon, {FontAwesomeV6IconProps} from '@/fontAwesomeV6Icon';
+import Link, {LinkProps} from '@/link';
+import {Heading5, BodyThreeText} from '@/typography';
+
+import moduleStyles from './iconHighlight.module.scss';
+
+export const ICON_HIGHLIGHT_DEFAULT_ICON = 'smile';
+
+export type IconHighlightLinkProps = LinkProps & {
+  key: Key;
+  external?: boolean;
+};
+
+export interface IconHighlightProps extends HTMLAttributes<HTMLElement> {
+  /** IconHighlight heading */
+  heading: string | ReactNode;
+  /** IconHighlight content */
+  text: string | ReactNode;
+  /** IconHighlight icon */
+  icon?: FontAwesomeV6IconProps;
```

Comment:
We may want to make icon required too since it's a main feature of this component. cc @moshebaricdo